src/styles/plugin.scss

```diff
@@ -43,6 +43,10 @@
   }
 }
 
+.thebelab-cell canvas {
+  max-width: unset;
+}
+
```

Comment:
Please add a comment here explaining that this is specifically for allowing ipyleaflet's canvas to work.

You could also check to see if the ipyleaflet canvas uses a specific class name or id name. If so you could make this unset only for ipyleaflet, ensuring it doesn't affect other packages.

Reply:
Done.